src/Dialogs/ListPicker.cpp

```diff
@@ -22,129 +22,11 @@ Copyright_License {
 */
 
 #include "Dialogs/ListPicker.hpp"
-#include "HelpDialog.hpp"
-#include "WidgetDialog.hpp"
-#include "Widget/ListWidget.hpp"
-#include "Widget/TextWidget.hpp"
-#include "Widget/TwoWidgets.hpp"
-#include "UIGlobals.hpp"
-#include "Language/Language.hpp"
-#include "Event/Timer.hpp"
-#include "Event/PeriodicTimer.hpp"
 
 #include <cassert>
 
 static constexpr int HELP = 100;
 
-class ListPickerWidget : public ListWidget, public ActionListener {
```

Comment:
Why delete this?

Reply:
This has been moved to `ListPicker.hpp` so that the `MultiFilePicker` can use it by including that file. If you want I can also move it to its own file.

Reply:
It is impossible to review a commit which moves and modifies code at the same time.